TracerLib/TracerImpl/Tracer.cs

```diff
@@ -49,14 +49,21 @@ public void StartTrace()
         public void StopTrace()
         {
             var st = new StackTrace();
-            var invokationFrame = (st.GetFrames())[1];
+            var invokationFrame = (st.GetFrames())?[1];
             var info = _methodsStack.Pop().Data;
-            info.ClassName = invokationFrame.GetMethod().DeclaringType.Name;
-            info.MethodName = invokationFrame.GetMethod().Name;
-            info.ParamCountInMethod = invokationFrame
-                .GetMethod()
-                .GetParameters()
-                .Count();
+            if (invokationFrame != null)
+            {
+                var declaringType = invokationFrame.GetMethod().DeclaringType;
+                if (declaringType != null)
```

Comment:
Maybe it will be better:
            var invokationFrame = (st.GetFrames())?[1];
            var declaringType = invokationFrame?.GetMethod().DeclaringType;
            if (invokationFrame != null && declaringType != null)
            { ... }